bool dump_to_file
---
float64[] recorded_sound
